src/components/Labels/Labels.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import cx from 'classnames';
+
+import './Labels.scss';
+
+export const CN = 'label';
+
+const Labels = (props) => {
+  const { blogItems, getLabel } = props;
+  const listOfLabels = blogItems?.map((obj) => obj.labels)?.flat()?.sort();
+  const countedLabels = {};
+
+  listOfLabels && listOfLabels.forEach((label) => {
+    if (!countedLabels[label]) {
+      countedLabels[label] = 1;
+    } else {
+      countedLabels[label] += 1;
+    }
+  });
+
+  const maxLabelCount = Math.max(...Object.values(countedLabels));
+
+  const labelElemToRender = (elem, fontSize) => (<span className={`${CN}__item`} style={{ fontSize: `${fontSize}rem` }}>{elem}</span>);
+
+  const divideInto = (num) => Math.round(maxLabelCount / num);
+
+  const formLableSize = () => {
+    const fontSizeforLargeEl = 1.6;
```

Comment:
Maybe better to move this constants out of function or even out of Label function?